### ISSUE TITLE
On the issue of encoding selection when signing with sign

### ISSUE THREAD

**Issue by yuwaixingtian** [paragonie/easy-ecc#10]
```php
$ecc = new EasyECC('K256');

$alice_sk = $ecc -> generatePrivateKey();

$alice_pk = $alice_sk -> getPublicKey() -> toString();

$pk = bin2hex($alice_pk);
$sign_dat = $ecc -> sign($data, $alice_sk, false);//I would like to know if you have a way to encode this place as sha256?
````

Because I observed the project and found the sign method in EasyECC.php：
```php
$kGen = new HedgedRandomNumberGenerator(
        EccFactory::getAdapter(),
        $privateKey,
       $hash,
      'sha384'
 ); 
```
It seems to be fixedly designated as sha384

**Comment by paragonie-security** [comment]
Oh, that's a good point.

**Comment by paragonie-security** [comment]
Fixed in `v1.2.0`

Just update to the latest version and your code will use the correct hash function for your ECDSA algorithm (SHA256 for secp256k1).
